app/src/main/res/values/strings.xml

```diff
@@ -9,9 +9,9 @@
     <string name="location">Location</string>
     <string name="feeling">Feeling</string>
     <string name="this_was_at">This was at</string>
-    <string name="month">mm</string>
-    <string name="year">yyyy</string>
-    <string name="day">dd</string>
+    <string name="month">month</string>
+    <string name="year">year</string>

```

Comment:
This is wrong. You should make sure you still have the MM,YYYY,DD strings and create new strings with different names.

Reply:
✅ correcting the strings